src/ru/skypro/EmployeeService.java

```diff
@@ -0,0 +1,229 @@
+package ru.skypro;
+
+public class EmployeeService {
+
+
+    public static boolean isNullEmployee(Employee employee) {
+        if (employee != null) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
такие вещи можно сокращать до
```suggestion
        return employee != null;
```
при этом имя метода и его логика не сходятся, метод спрашивает, null ли переданный работник, а мы отвечаем true на not null значение